test/e2e/framework/metrics_util.go

```diff
@@ -521,6 +528,53 @@ func VerifySchedulerLatency(c clientset.Interface) (*SchedulingMetrics, error) {
 	return latency, nil
 }
 
+func ResetSchedulerLatencyMetrics(c clientset.Interface) error {
```

Comment:
I see some overlap of this function with `getSchedulingLatency()`. Can we somehow unify them?